docs/modules/concepts/pages/storage/users.adoc

```diff
@@ -0,0 +1,59 @@
+= User Directory
+:navtitle: Users
+
+== Overview
+
+Whereas xref:storage/mailbox.adoc[mailboxes] are clearly a core component of
+any email system, user management is a lighter shade of gray.
+
+On the one hand, mailboxes ought to match up with actual users, so there is
+a clear relationship between mailbox management and user management.
+Because of this relationship, it is not at all strange to include a user directory
+directly within an email system.
+
+However, on the other hand email is only one of many applications that require
+user management. It is therefore also completely reasonable to use some kind of
+centralized user directory, and ensure that mailboxes somehow synchronize with
+this external system.
```

Comment:
`and ensure that mailboxes somehow synchronize with
this external system.`

This somehow you are looking for is LDAP as a UsersRepository implementation ?

Reply:
Of course we need to mention that JAMES will "read" the LDAP anuary, not modify it.

Reply:
Again, my intent here is to be more general. How James works would be left for a different section. That is my thought now, anyway. We'll see how that actually works out.

Reply:
> Again, my intent here is to be more general. 

Overall +1. Sorry I failed at extracting the general part of this statement, but I guess there's still something to get out of it.

Reply:
Probably it needs to be explained better. The idea is to make a gentle introduction into the various concepts so that when we later explain how James works, it won't seem so alien. At least, that is the idea. Since this is the first iteration, like many parts could be greatly improved.

Reply:
I'm going to leave this as is for now. I will be doing another review later.